### DISCUSSION TITLE
Accessing a directory that requires super user privileges in Linux

### DISCUSSION THREAD

**Discussion by bmcgonag** [neutralinojs/neutralinojs#1155] [General]
I have a little app, and I want to allow the user to scan a directory that requires super user privileges.  The user has to allow it and provide their super user password of course. But the filesystem api doesn't seem to have any way to do this.  Am I missing something, or is there a way to do it that isn't documented?

Thanks,

**Comment by shalithasuranga** [comment]
Hey. thanks for posting this discussion thread. The file system API should be able to read/write super user content once the user runs the app in the elevated mode (In Unix/Linux, with `sudo`). Did you already try that approach and seek another workaround to improve user-friendliness?

**Reply by bmcgonag** [reply]
I was hoping to only access something with "sudo" as needed.  I don't feel good about asking the user to run the application in sudo mode, as it doesn't generally need access for most things, but only for a couple of specific instances.
